suite-native/module-dev-utils/src/screens/DevUtilsScreen.tsx

```diff
@@ -48,6 +48,7 @@ export const DevUtilsScreen = ({
 
     const handleResetStorage = useCallback(() => {
         persistor.purge().then(() => NativeModules.DevSettings.reload());
+        purgeStorageEncryptionKey();
```

Comment:
You should probably create only one function like `purgeStorage` in storage package, that will do both these things at one time and use it here.

Reply:
done in de5504964